### ISSUE TITLE
FXIOS-322 ⁃ [a11y] Can't dismiss bookmark actions screen with VoiceOver

### ISSUE THREAD

**Issue by jcsteh** [mozilla-mobile/firefox-ios#6345]
STR:

1. Ensure you have at least one bookmark.
2. Tap Menu, then Your Library.
3. Turn on VoiceOver.
4. Tap a bookmark, then triple tap to bring up the bookmark actions screen.
5. Two finger scrub to go back.
    - Expected: Nothing happens.
    - Actual: The bookmark actions screen should be dismissed, returning you to the library screen.

There's no back or dismiss button either, so at present, there's no way for a VoiceOver user to dismiss this screen at all without performing an action. I think not having an explicit button is fine if the back (2 finger scrub) gesture works.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-322)

**Comment by jcsteh** [comment]
P1 for accessibility because this is functionality that a VoiceOver user can't access at all.

**Comment by data-sync-user** [comment]
➤ Daniela Arcese commented:

Fixed in FXIOS-1490

**Comment by SimonBasca** [comment]
Verified fix on v34.0 (4680) and main 493d272cca.
